css/alumni.sass

```diff
@@ -5,15 +5,27 @@
 #collegeName 
     font-size: 30px
     color: black
+
 #collegeImg 
     max-width: 150px
     max-height: auto
     display: block
     margin-left: auto
     margin-right: auto
- 
-    
+
+.li-quotes
+    list-style: none
```

Comment:
I would advocate that instead we simply remove the ul/li tags and target `blockquote` elements directly. At the very least, this class should be called `quote` or something, as you can just say `li.quote` to specify li elements—no need for including the tag name in the class

Reply:
done 👍